system/security/LdapSecurity/permissions.cpp

```diff
@@ -77,10 +77,10 @@ unsigned PermissionProcessor::ldap2sec(unsigned ldapperm)
     if(ldapperm & 0x28)
         permission |= SecAccess_Write;
```

Comment:
ok, due to this cascading set of bitwise ops I can see why permission was kept as an unsigned, but now I'm wondering if this is sound logic... are we guaranteed to have an enum representation for all the possible values? we could end up with permissions = 0 | SecAccess_Access | SecAccess_Write, then we do a cast to SecAccessFlags line 80...  Is there an enum value for SecAccess_Access | SecAccess_Write ? and if not, what will happen when you cast?